src/qt/pivx/settings/forms/settingsfaqwidget.ui

```diff
@@ -0,0 +1,1365 @@
+                      </property>
+                      <property name="alignment">
+                       <set>Qt::AlignCenter</set>
+                      </property>
+                     </widget>
+                    </item>
+                    <item>
+                     <layout class="QVBoxLayout" name="verticalLayoutpage_7">
+                      <property name="leftMargin">
+                       <number>20</number>
+                      </property>
+                      <item>
+                       <widget class="QLabel" name="labelSubtitle7">
+                        <property name="text">
+                         <string>What is PIVX</string>
+                        </property>
+                        <property name="wordWrap">
+                         <bool>true</bool>
+                        </property>
+                       </widget>
+                      </item>
+                      <item>
+                       <spacer name="verticalSpacer_page_7">
+                        <property name="orientation">
+                         <enum>Qt::Vertical</enum>
+                        </property>
+                        <property name="sizeType">
+                         <enum>QSizePolicy::Fixed</enum>
+                        </property>
+                        <property name="sizeHint" stdset="0">
+                         <size>
+                          <width>20</width>
+                          <height>30</height>
+                         </size>
+                        </property>
+                       </spacer>
+                      </item>
+                      <item>
+                       <widget class="QLabel" name="labelContent7">
+                        <property name="text">
+                         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To Stake PIVX:&lt;/p&gt;&lt;p&gt;    1) Make Sure your wallet is completely synchronized and you are using the latest release.&lt;/p&gt;&lt;p&gt;    2) You must have a balance of PIVX with a minimum of 101 confirmations.&lt;/p&gt;&lt;p&gt;    3) Your wallet must stay online and be Unlocked for Anonymization and Staking Purposes.&lt;/p&gt;&lt;p&gt;    4)Once all those steps are followed staking should be enabled.&lt;/p&gt;&lt;p&gt;    5)You can see the status of staking in the wallet by viewing the bottom right of the wallet interface. There should be a green light indicating it is staking. If you are using the Daemon then you can run getstakingstatus to confirm that staking is active.&lt;/p&gt;&lt;p&gt;To Stake PIVX:&lt;/p&gt;&lt;p&gt;    1) Make sure your wallet is completely synchronized and you are using the latest release.&lt;/p&gt;&lt;p&gt;    2) Your newly minted or existing zPIV balance must have a minimum of 200 confirmations.&lt;/p&gt;&lt;p&gt;    3) Your wallet must stay online and be Unlocked for Anonymization and Staking Purposes. Staking should now be enabled.&lt;/p&gt;&lt;p/&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
lowercase "Sure"
lowercase both instances of "Unlocked for Anonymization and Staking Purposes"
Second section, I suspect should be "To Stake zPiv".  It currently has two sections, both "To Stake PIVX"

Reply:
done 👍

Reply:
second instance isn't lowercased yet.  This can stay resolved; as I have another pass through coming up now that I'm seeing it in app instead of in code.